include/mbedtls/md.h

```diff
@@ -149,19 +152,13 @@ typedef enum {
     MBEDTLS_MD_SHA384,    /**< The SHA-384 message digest. */
     MBEDTLS_MD_SHA512,    /**< The SHA-512 message digest. */
     MBEDTLS_MD_RIPEMD160, /**< The RIPEMD-160 message digest. */
+    MBEDTLS_MD_SHA3_224,    /**< The SHA3-224 message digest. */
+    MBEDTLS_MD_SHA3_256,    /**< The SHA3-256 message digest. */
+    MBEDTLS_MD_SHA3_384,    /**< The SHA3-384 message digest. */
+    MBEDTLS_MD_SHA3_512,    /**< The SHA3-512 message digest. */
 } mbedtls_md_type_t;
 
-/* Note: this should always be >= PSA_HASH_MAX_SIZE
```

Comment:
Why did you remove this entire note? I think it is still valid (except for the last sentence) and useful.

Reply:
Sorry, I made a mistake in the merge.

Reply:
No worries, happens!